File: 1_course/1_week/1_sum_of_two_digits/c/APlusB.c
/**
 *
 * C implementation to sum two integers
 *
 * (c) Copyright 2019 Clayton J. Wong ( http://www.claytonjwong.com )
 *
 **/

#include <stdio.h>

int sum_of_two_digits( int first, int second ){
    return first + second;
}

int main(){
    int a = 0, b = 0;
    scanf( "%d%d", &a, &b );
    printf( "%d\n", sum_of_two_digits( a, b ));
    return 0;
}

File: 1_course/week1/1_sum_of_two_digits/java/src/APlusB.java
/**
 *
 * Java implementation to sum two integers
 *
 * (c) Copyright 2019 Clayton J. Wong ( http://www.claytonjwong.com )
 *
 **/

import java.util.Scanner;

class APlusB {

    static int sumOfTwoDigits( int first, int second ){
        return first + second;
    }

    public static void main( String[] args ){
        Scanner s = new Scanner( System.in );
        int a = s.nextInt();
        int b = s.nextInt();
        System.out.println( sumOfTwoDigits( a, b ) );
    }
}
